src/ml/neural_net/mps_layer_conv_padding.m

```diff
@@ -0,0 +1,57 @@
+#include <ml/neural_net/mps_layer_conv_padding.h>
+
+@implementation ConvolutionPadding
+
++ (BOOL) supportsSecureCoding {
+  return true;
```

Comment:
Nit: for Objective C code, use YES or NO instead of true or false.

Reply:
Changed!